docs/gestor_dependencias.md

```diff
@@ -0,0 +1,36 @@
+# Elección del gestor de dependencias
+
+Es una decisión primordial, ya que un gestor de dependencias adecuado nos permitirá un desarrollo lo más optimizado.
+#### Aspectos a tener en cuenta:
+
+*  Comunidad
+* Mantenimiento (Freshness)
+* Velocidad
+      
+Las opciones principales que se han investigado son:
+
+## NPM
+
+- Gestor por defecto y más usado (mayor comunidad, enfoque simple y estar bien documentado).
+- Actualizaciones constantes.
+- Funcionalidad básica de gestor de dependencias.
+- No destaca su velocidad
+
+## YARN
+
+- Tiene una comunidad muy activa y en su github tienen muchos `issues` con los que se pueden solventar errores y problemas frecuentes.
+- Se mantiene de forma continua (`release` reciente y con nuevos commits en la rama principal de desarrollo), garantizando de momento unas condiciones fiables.
+- Muy veloz y seguro (se hizo pensando en estas características)
+
+## PNPM
+
+- Comunidad grande, cuenta con muchos foros, guías y tutoriales
```

Comment:
Pero ¿activa? ¿Es mejor que sea grande o que sea activa?

Lee: no pongáis criterios irrelevantes y que no se puedan evaluar de forma objetiva.